CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+# 0.33.0-rc1 (2020-09-27)
+
+This is a release candidate to preview the changes in the next official release and may not be stable. Welcome any questions or suggestions.
+
+[CKB v0.35.1](https://github.com/nervosnetwork/ckb/releases/tag/v0.35.1) was released on Sept. 14th, 2020. This version of 
+CKB node is now bundled and preconfigured in Neuron. 
+
+Note: There will be a data migration for the new bundled node [CKB v0.35.1](https://github.com/nervosnetwork/ckb/releases/tag/v0.35.1), which may take several minutes to boot up for the first time after upgraded from the earlier versions.
+
+### Refactor
+
+* Upgrade from Electron 7 to Electron 9.
+* Refactor the wallet address storage to be based on an SQLite table rather than a JSON file.
+* Deprecate LevelDB and use SQLite instead.
+* Refactor send component.
+
+### New features
+
+* Change the confirmation block height from 300 to 24.
```

Comment:
Change => Reduce

Reply:
> Reduce

OK

Reply:
I would suggest this point `Reduce the confirmation block height from 300 to 24.` into refactor section as it is not quite a new feature.